lib/mp4/transmuxer.js

```diff
@@ -140,38 +213,45 @@ VideoSegmentStream = function(track) {
 
   delete track.minPTS;
 
-  this.push = function(data) {
-    collectDtsInfo(track, data);
+  this.gopCache = [];
```

Comment:
Do you foresee a need for an outside part to manipulate the cache other than clearing it?